src/frontend/components/SelectContent/route.ts

```diff
@@ -0,0 +1,6 @@
+/**
+ * Route for the `<SelectContent />` component.
+ */
+export const SELECT_CONTENT_ROUTE = () => {
+  return '/select/';
+};
```

Comment:
Why did you use a function here?
```suggestion
export const SELECT_CONTENT_ROUTE = '/select/'
```

Reply:
Because other routes uses functions, like https://github.com/openfun/marsha/blob/master/src/frontend/components/RedirectOnLoad/route.ts

Reply:
Oh right! Homogeneity is good !

Reply:
I simplified it though.